packages/amplify_api_plugin_interface/lib/src/GraphQL/GraphQLResponse.dart

```diff
@@ -13,12 +13,11 @@
  * permissions and limitations under the License.
  */
 
-import 'package:flutter/foundation.dart';
 import '../types.dart';
 
 class GraphQLResponse<T> {
   T data;
-  List<GraphQLResponseError> errors = [];
+  List<GraphQLResponseError>? errors = [];
```

Comment:
Is this accurate? Didn't we already initialize it to be non-null. Are we saying that the caller can initialize it to null and that's expected?

Reply:
Good point.  I think this should be switched to just nullable and no initialization.

@Ashish-Nanda perhaps you can provide some insight here on how this class is used.  Based on the usage I see, it appears this field can be nullable and doesn't need the blank list [] initialization?